src/go/rpk/pkg/config/rpk_yaml_test.go

```diff
@@ -25,7 +25,7 @@ func TestRpkYamlVersion(t *testing.T) {
 	shastr := hex.EncodeToString(sha[:])
 
 	const (
-		v5sha = "74f0b320006aac52cb75af3d5c658f81decf3e40fe1e92f3eb1d59f5c47ab5f9" // 24-04-29
+		v5sha = "f3d394e88ba4a6a668d15b74208f32d71c82f8a0e9b6e6445defcdd64df1e37f" // 24-07-24
```

Comment:
I'd rename this variable to `expsha`; my original thinking was that we'd keep the old sha's around for history, but we haven't and it's not necessary.

Reply:
Fixed in https://github.com/redpanda-data/redpanda/pull/23472/commits/225b550e4de8d55e6def7966cc17a338a7c954e2